src/com/musala/xml/model/Teacher.java

```diff
@@ -0,0 +1,36 @@
+package com.musala.xml.model;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+
+public class Teacher {
+    public Teacher() {
+    }
+    private String name;
+    private int age;
+    private String referenceNumber;
+
+    public int getAge() {
+        return age;
+    }
+
+    public String getName() {
```

Comment:
In general, for all classes, make a decision about the getter/setter structure. Either declare all getters (in alphabetical order) followed by all setters (again in alphabetical order), or declare getter/setter pairs in alphabetical order. Apply your decision for all classes (consistency reasons).